numpy/random/mtrand/mtrand.pyx

```diff
@@ -4460,8 +4460,8 @@ cdef class RandomState:
 
         A loaded dice is more likely to land on number 6:
 
-        >>> np.random.multinomial(100, [1/7.]*5)
-        array([13, 16, 13, 16, 42])
+        >>> np.random.multinomial(100, [1/7.]*6)
```

Comment:
Yes, examples are not deterministic (are they anywhere in numpy?). This is an obscure feature, I agree --- and I suspect the docstrings was like this by design, specifically to illustrate the convention. Also, the convention *should* be illustrated I think, esp given that eg R decides instead to scale the probabilities by dividing by the sum.
In  https://github.com/scipy/scipy/pull/5410 I wrote it a bit more explicitly,https://github.com/scipy/scipy/pull/5410/files#diff-3f67e7fdb1ce6a44c0b49df2da9889c5R2290. Can do it here as well if you want me to.